music_assistant/server/providers/snapcast/__init__.py

```diff
@@ -668,8 +684,9 @@ async def _builtin_server_runner(self) -> None:
             "--http.port=1780",
             f"--http.doc_root={SNAPWEB_DIR}",
             "--tcp.enabled=true",
-            "--tcp.port=1705",
-            f"--stream.buffer={self._snapcast_server_control_port}",
+            f"--tcp.port={self._snapcast_server_control_port}",
+            f"--stream.buffer={self._snapcast_server_buffer_size}",
```

Comment:
The port control is only intended for an external server, does it change anything to allow the user to change the port of the embedded server? Doesn't it just add complexity?

Reply:
I think when the external server is disabled this value is just set to `DEFAULT_SNAPSERVER_PORT` as in L291-293?

Reply:
You are right, this is ok

Reply:
I don't know how to do it, I do it from vscode with the extension for github